content/spin/v2/language-support-overview.md

```diff
@@ -59,7 +59,7 @@ This page contains information about language support for Spin features:
 
 {{ startTab "Python"}}
 
-[Visit the Spin SDK reference documentation.](https://fermyon.github.io/spin-python-sdk/)
+**[📄 Visit the Python Spin SDK reference documentation](https://fermyon.github.io/spin-python-sdk/) to see specific modules, functions, variables and syntax relating to the following Python SDK.**
```

Comment:
```suggestion
**[📄 Visit the Python Spin SDK reference documentation](https://fermyon.github.io/spin-python-sdk/) to see the modules, types, functions, and variables in the Python SDK.**
```

(Although personally I find that extra clause a bit redundant.  It's kind of "visit the Python SDK docs to see what's in the Python SDK."  And that enumerating the various kinds of things detracts from the message rather than adding to it.  But YMMV!)